src/BuiltInTools/dotnet-watch/HotReload/HotReload.cs

```diff
@@ -26,12 +26,20 @@ public async ValueTask InitializeAsync(DotNetWatchContext dotNetWatchContext, Ca
 
         public async ValueTask<bool> TryHandleFileChange(DotNetWatchContext context, FileItem file, CancellationToken cancellationToken)
         {
-            if (await _staticFileHandler.TryHandleFileChange(context, file, cancellationToken))
+            HotReladEventSource.Log.HotReloadStaticFileStart(file.FilePath);
```

Comment:
Is it interesting to try and track each of these individually? Typically only one of these would work (it's either a css file and we tell the browser to refresh or it's a .cs / .razor file and we have to produce an EnC delta and apply it).